scout/server/blueprints/managed_variants/views.py

```diff
@@ -29,21 +29,20 @@ def upload_managed_variants():
 
     csv_file = request.files["csv_file"]
     content = csv_file.stream.read()
-    lines = None
     try:
         if b"\n" in content:
             lines = content.decode("utf-8-sig", "ignore").split("\n")
         else:
             lines = content.decode("windows-1252").split("\r")
+
+        result = controllers.upload_managed_variants(store, lines, institutes, current_user._id)
```

Comment:
The code gets a bit complicated with the nested try-excepts - though perfectly valid. And the readability goes down slightly - on a quick first read I got the impression we could end up trying to load with a faulty parse. 

This would be ok with a big gain, I might just be a bit slow from thinking about too many other projects, but I don't quite get what you are trying to solve here? The flash would happen on exception, so both trigger just fine and given that the we arrived from the managed variants page, we should end up back there?

With main:
![Screenshot 2024-04-30 at 11 05 47](https://github.com/Clinical-Genomics/scout/assets/758570/e0de663b-ef38-4a7c-94ed-8a8e3752fed2)

With PR:
![Screenshot 2024-04-30 at 11 02 59](https://github.com/Clinical-Genomics/scout/assets/758570/bae1441f-780f-4f78-babb-05158ad2da06)